workflow/log-monitoring/transactions.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Transforming Logs into Transactions
+sidebarTitle: Transactions
+---
+
+Transactions traverse across your system and aggregate your log data with other data that exists outside of the log you are already filtering. This feature is useful when you need to identify broader patterns within your infrastructure that are related to the log you are actively filtering. 
+
+Transactions work by compiling indexing logs based on occurrences of a series of actions, like a user's session or a request handled across your microservice architecture. Examples of transactions range from actions within a mobile app, to function calls and CPU instructions.
+
+# Transaction Measures
+
+Measures are additional ways for you to filter your transactions:
+
+| __Measure__          | __Description__ |
+| :----------          | :-------------- |
+| __Count__            | Tracks the total number of times the aggregated metric has been fired |
+| __Max__              | Displays the maximum value of the aggregated metric |
+| __Most Frequent__    | Identifies the most frequent value of the aggregated metric |
+| __Where__            | Adds additional aggregation layer to query |         
+| __Contains__         | Filters log entry based on whether the specified field contains a particular string or subpattern |
+| __Does Not Contain__ | Filters log entry based on whether the specified field __does not__ contains a particular string or subpattern |
+| __Greater Than__     | Aggregates metrics that are greater than a specified threshold or reference point |
+| __Smaller Than__     | Aggregates metrics that are less than a specified threshold or reference point |
+
+# Create a Transaction
+
+Navigate to the logs page, select the Transactions tab, choose your desired aggregation metric, and add one or multiple measures to filter your data even further. 
```

Comment:
I would remove "to filter your data even further" since it's not filtering. We could sub it out with something like "and add one or multiple measures to better understand your logs."